bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -379,4 +379,4 @@ spec:
   provider:
     name: Red Hat
   replaces: grafana-operator.v4.7.1
```

Comment:
I apparently miss wrote this the last time. 
4.8.0 replaces 4.7.1.

I guess I manually fixed that in upstream OLM.